refine.bat

```diff
@@ -141,11 +131,25 @@ goto shift2loop
 set OPTS=%OPTS% -Dcom.sun.management.jmxremote
 goto shift2loop
 
+:arg-c
+set REFINE_INI_PATH=%~2
+goto shift2loop
+
 :shift2loop
 shift
 shift
 goto loop
 
+:readIniFile
+
+rem --- Read ini file -----------------------------------------------
+
+if "%REFINE_INI_PATH%" == "" set REFINE_INI_PATH=refine.ini
+echo Using %REFINE_INI_PATH% for configuration
+for /f "tokens=1,* delims==" %%a in (%REFINE_INI_PATH%) do (
```

Comment:
You'll need to handle a **file not found**.  Perhaps quickly fail here and echo the error `The system cannot find the file C:\Users\thadg\.sonarlint\refine.ini.`?

My invalid path fail test `refine /c C:\Users\thadg\.sonarlint\refine.ini` still progressed and tried to launch Refine:
```
E:\GitHubRepos\OpenRefine>goto loop

E:\GitHubRepos\OpenRefine>if """" == """" goto readIniFile

E:\GitHubRepos\OpenRefine>rem --- Read ini file -----------------------------------------------

E:\GitHubRepos\OpenRefine>if "C:\Users\thadg\.sonarlint\refine.ini" == "" set REFINE_INI_PATH=refine.ini

E:\GitHubRepos\OpenRefine>echo Using C:\Users\thadg\.sonarlint\refine.ini for configuration
Using C:\Users\thadg\.sonarlint\refine.ini for configuration

E:\GitHubRepos\OpenRefine>for /F "tokens=1,* delims==" %a in (C:\Users\thadg\.sonarlint\refine.ini) do (set %a=%b )
The system cannot find the file C:\Users\thadg\.sonarlint\refine.ini.

E:\GitHubRepos\OpenRefine>rem --- Fold in Environment Vars --------------------------------------------

E:\GitHubRepos\OpenRefine>if not "" == "" goto gotJavaOptions

E:\GitHubRepos\OpenRefine>set JAVA_OPTIONS=

E:\GitHubRepos\OpenRefine>set OPTS=

E:\GitHubRepos\OpenRefine>if not "" == "" goto gotMemory

E:\GitHubRepos\OpenRefine>set REFINE_MEMORY=1024M

E:\GitHubRepos\OpenRefine>if not "" == "" goto gotMemory
```

Reply:
Something? https://ss64.com/nt/syntax-error.html
Or? https://stackoverflow.com/questions/53654565/batch-exception-handling-using-a-for-loop-with-multiline-in-running-the-commands

Reply:
Looks like from reading around that `in` is special within for loops and a launches cmd.exe separate process outside of batch-file context.  See the comment in 2nd link https://stackoverflow.com/a/53658484

Reply:
and https://superuser.com/a/615141

Reply:
@thadguidry I just added "if not exist" and it is working fine. So the flow now is, if /**c** is not given it will take refine.ini from the root folder, if given it will take that. And if any of these files does not exist it will throw exception and exit.

Reply:
@tejasrbhat tested and works great now!